cmd/juju/model/configcommand.go

```diff
@@ -155,7 +221,10 @@ func (c *configCommand) handleZeroArgs() error {
 func (c *configCommand) handleOneArg(arg string) error {
 	// We may have a single config.yaml file
 	_, err := os.Stat(arg)
-	if err == nil || strings.Contains(arg, "=") {
+	if err == nil {
```

Comment:
Can we add support for passing '-' instead of a yaml file to read from STDIN? This, combined with my other comment about dropping `agent-version` would make it easy to do something like:

```
juju model-config -m stc_model --format=yaml | juju model-config -m dst_model  -
```

Reply:
Well for now I'm happy with the workaround of a `tmpfile`... if people want the stdin to stdout then I'll happily add it.